### PR TITLE
Fix triple protocol thread name

### PR DESCRIPTION
**Pull request by CrazyHZM**
## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

**Review by EarthChen — APPROVED** [review]
LGTM

File: dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java [hunk 1]
```diff
@@ -83,8 +84,14 @@ public synchronized ExecutorService createExecutorIfAbsent(URL url) {
...
         Integer portKey = CONSUMER_SIDE.equalsIgnoreCase(url.getParameter(SIDE_KEY)) ? Integer.MAX_VALUE : url.getPort();
+
+        String protocol = url.getProtocol();
+        if (StringUtils.isEmpty(protocol)) {
+            protocol = DEFAULT_PROTOCOL;
+        }
+
         if (url.getParameter(THREAD_NAME_KEY) == null) {
-            url = url.putAttribute(THREAD_NAME_KEY, "Dubbo-protocol-" + portKey);
+            url = url.putAttribute(THREAD_NAME_KEY, protocol + "-protocol-" + portKey);
```
[the remark this answers is not in the document]
Comment (CrazyHZM) [reply dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java:94]: We can't move the logic over here, it's not only affecting triple.

[the remark this answers is not in the document]
Comment (BurningCN) [reply dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java:94]: 大佬好，不知道我有没有表达清楚。我看到88~91行这段获取的protocol变量只有在threadNameKey为空的时候才使用到，如果93行条件不满足，即url已经有threadNameKey了，那么88~91行获取protocol的操作是不是就没有意义了。

[the remark this answers is not in the document]
Comment (CrazyHZM) [reply dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java:94]: 理解了，不过没什么影响，这里并不会被频繁调用。

**Pull request merged**